sdk/monitor/Azure.Monitor.Ingestion/tests/IngestionClientTest.cs

```diff
@@ -0,0 +1,227 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.Identity;
+using NUnit.Framework;
+using Azure.Monitor.Query;
+using Azure.Monitor.Query.Models;
+using System.Threading.Tasks;
+using System.Linq;
+
+namespace Azure.Monitor.Ingestion.Tests
+{
+    public class IngestionClientTest : RecordedTestBase<IngestionClientTestEnvironment>
+    {
+        public IngestionClientTest(bool isAsync) : base(isAsync)
+        {
+        }
+
+        /* please refer to https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/template/Azure.Template/tests/TemplateClientLiveTests.cs to write tests. */
+
+        [RecordedTest]
+        public void TestOperation()
+        {
+            Assert.IsTrue(true);
+        }
+
+        #region Helpers
+
+        private static BinaryData GetContentFromResponse(Response r)
+        {
+            // Workaround azure/azure-sdk-for-net#21048, which prevents .Content from working when dealing with responses
+            // from the playback system.
+
+            MemoryStream ms = new MemoryStream();
+            r.ContentStream.CopyTo(ms);
+            return new BinaryData(ms.ToArray());
+        }
+        #endregion
+
+        [Test]
+        public async Task ValidInputFromObjectAsJson()
+        {
+            ClientSecretCredential clientSecretCrendential = new ClientSecretCredential("<tenantId>", "<cliendId>",  "<clientSecret>");
+            var dcrImmutableId = "<dcrImmutableId>";
+            var dcrEndpoint = "https://nibhati-dce-ku6s.westus2-1.ingest.monitor.azure.com";
+            var currentTime = DateTimeOffset.UtcNow.ToString("O");
```

Comment:
Use `Recording.UtcNow` per https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/core/Azure.Core.TestFramework#misc instead so you won't have to change this when you switch to recordings